client/src/components/Pages/RegElement/RegElementForm/RegElementForm.tsx

```diff
@@ -1,25 +1,14 @@
 import {
-  Button,
   FormControl,
   InputLabel,
   MenuItem,
   Select,
   Typography,
```

Comment:
You could probably just move these remnant reg element files into the Structure subdirectory to match the other items there